src/styles/styles.scss

```diff
@@ -149,6 +154,27 @@ a {
   }
 }
 
+.feedback-row {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+
+  &--label {
+    width: 60%;
```

Comment:
These percentages make perfect sense to me so I'm not sure why they're not always working, but when I run this locally the progress bars are losing their left-alignment when the display goes below 1300-ish pixels in width. We could try setting a `min-width` to see if that would help? Or switch to a grid for guaranteed alignment but that might be a pain at this point.